stream-dbs-clients/pom.xml

```diff
@@ -234,6 +234,18 @@
                             <modelPackage>com.backbase.dbs.accesscontrol.api.integration.v2.model</modelPackage>
                         </configuration>
                     </execution>
+                    <execution>
```

Comment:
perhaps we should remove from here the code generation for the v2 clients

Reply:
@uesleilima do you know where `api.integration.v2.model` is used? I can't find any usages in the Stream itself. Is it safe to remove `generate-accesscontrol-integration-api-code` execution completely?